### DISCUSSION TITLE
Embed stat card into form

### DISCUSSION THREAD

**Discussion by occoder** [h2oai/wave#635] [Q&A]
I'd like to group stat cards into form and tried to implement it using
`the_form = ui.form_card(box='1 1 4 -1', items=[ui.small_stat_card(title='demo', value='1000')])`
But it turned out reflecting message like
`This component could not be rendered.`
So the question is what is the appropriate way to make group of stat cards? 
Thanks

**Comment by lo5** [comment]
@occoder Currently it's not possible to include stats cards inside a form_card (or in general, include one card inside another). This will be addressed by #147.

Related: a `ui.stats()` component API will be out in the next release, which will support displaying simple label/value pairs inline in a form, via https://github.com/h2oai/wave/commit/3fd1956b8cca45817781d95be6c22fd42367ed55
